index.js

```diff
@@ -26,7 +26,6 @@ export { default as SearchAndSort } from './lib/SearchAndSort';
 export { default as SearchAndSortQuery } from './lib/SearchAndSort/SearchAndSortQuery';
 export { default as SearchAndSortNoResultsMessage } from './lib/SearchAndSort/components/NoResultsMessage';
 export { default as SearchAndSortResetButton } from './lib/SearchAndSort/components/ResetButton';
-export { default as SearchAndSortSearchButton } from './lib/SearchAndSort/components/SearchButton';
```

Comment:
Is this on purpose? This is a breaking change for modules that use `SearchAndSortQuery` like the ERM modules. [Here's our usage.](https://github.com/folio-org/ui-agreements/blob/master/src/components/views/Agreements.js#L23)

Reply:
Other users are at least:
- ui-licenses
- ui-local-kb-admin
- ui-plugin-find-[user/license/agreement]

Reply:
Hey @doytch 

This work is still WIP and I was going to check up on that part.  But thanks for letting me know! I'll leave it in there so nothing breaks.

I assume that the changes for the UX in this PR will have to be reflected across all apps eventually.

It's easily achieved with the `<PaneHeaderIconButton>` from stripes-components. Note that the collapse button is in the filter pane now and the expand button will be located in the results pane.

Reply:
Phew! 😅 

And gotcha...yeah, figured that we'd eventually have to update the other apps to reflect these changes. I think this is one of those places where having separated everything due to `SearchAndSortQuery` can hurt us at the moment and we'd benefit from a few more `SearchAndSortUIWidget` components to smooth over transitions.

Reply:
@doytch I re-added the search pane toggle component 👍